<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f5dc;</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <property name="verticalSpacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lblUser">
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Usuario:   </string>
      </property>
      <property name="buddy">
       <cstring>&amp;Usuario:</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leUser">
      <property name="toolTip">
       <string>Introduce tu nombre de usuario</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="maxLength">
       <number>24</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Usuario</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblContra">
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Contraseña:  </string>
      </property>
      <property name="buddy">
       <cstring>&amp;Contraseña:</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lePass">
      <property name="toolTip">
       <string>Introduce tu contraseña</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="maxLength">
       <number>24</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Contraseña</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btnLogin">
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>Ingresar</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnRegistro">
        <property name="font">
         <font>
          <family>Dubai</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black
</string>
        </property>
        <property name="text">
         <string>Registrarse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMsg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblLogIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Inicia Sesión</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
